src/Command/DbDump.php

```diff
@@ -91,8 +70,17 @@ public function __construct(
     protected function configure()
     {
         $this->setName(self::NAME)
-            ->setDescription('Creates backup of database');
-
+            ->setDescription('Creates backups of databases');
+        $this->addArgument(
+            self::ARGUMENT_DATABASES,
+            InputArgument::IS_ARRAY,
+            sprintf(
+                'Databases to backup. Available values: %s or empty. By default will backup the databases'
+                . ' based on the databases configuration from the file <magento_root>/app/etc/env.php ',
+                implode(' ', DumpProcessor::DATABASE_MAP)
```

Comment:
Why not to leave IDs from env.php: default, checkout, sales?

Reply:
also, implode values through comma would be more readable

Reply:
> Why not to leave IDs from env.php: default, checkout, sales?

Because console commands of Magento use the part like 'sales'  and 'quote'​​to switch to split database mode:
```
magento setup:db-schema:split-sales --host="<checkout db host or ip>" --dbname="<name>" --username="<checkout db username>" --password="<password>"
magento setup:db-schema:split-quote --host="<checkout db host or ip>" --dbname="<name>" --username="<checkout db username>" --password="<password>"

```
Also, the same values ​​are used to enable split database mode on Magento Cloud:
```
stage:
  deploy:
    SPLIT_DB:
       - quote
       - sales
```

> also, implode values through comma would be more readable

Yes, but it can be confusing for the user because the values ​​are entered using a space as a separator